stable/elasticsearch/README.md

```diff
@@ -63,7 +63,7 @@ The following table lists the configurable parameters of the elasticsearch chart
 | ------------------------------------ | ------------------------------------------------------------------- | --------------------------------------------------- |
 | `appVersion`                         | Application Version (Elasticsearch)                                 | `6.6.1`                                             |
 | `image.repository`                   | Container image name                                                | `docker.elastic.co/elasticsearch/elasticsearch-oss` |
-| `image.tag`                          | Container image tag                                                 | `6.5.1`                                             |
+| `image.tag`                          | Container image tag                                                 | `6.5.4`                                             |
```

Comment:
This should be in a separate change.

Reply:
@rendhalver the PR was out-of-date. I've merged it with master. It should be OK now.